src/test/java/com/wikia/webdriver/testcases/communityheadertests/CommunityHeaderTests.java

```diff
@@ -0,0 +1,139 @@
+package com.wikia.webdriver.testcases.communityheadertests;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.oasis.CommunityHeader;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.oasis.MainPage;
+
+import junit.framework.Assert;
+import org.testng.annotations.Test;
+
+public class CommunityHeaderTests extends NewTestTemplate {
+
+  @Test(groups = {"CommunityHeaderTests"})
+  public void wordmarkShouldLinkToMainPage() {
+    MainPage mainPage = new CommunityHeader().clickWordmark();
+
+    Assert.assertTrue(mainPage.isMainPage());
+  }
+
+  @Test(groups = {"CommunityHeaderTests"})
+  public void wikiNameShouldLinkToMainPage() {
+    MainPage mainPage = new CommunityHeader().clickWikiName();
+
+    Assert.assertTrue(mainPage.isMainPage());
+  }
+
+  @Test(groups = {"CommunityHeaderTests"})
+  @Execute(asUser = User.ANONYMOUS)
+  public void testAnonWikiButtons() {
+    new CommunityHeader().clickAddNewPage();
+
+    Assert.assertTrue(driver.getCurrentUrl().contains("Special:CreatePage"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Special:CreatePage" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)